dev/Installer/src/help.cpp

```diff
@@ -0,0 +1,18 @@
+#include "pch.h"
+#include "help.h"
+
+void show_help()
+{
+    std::wcout << L"Installs the Microsoft Project Reunion framework." << std::endl;
```

Comment:
These should likely be localized.

Reply:
I've created #456 to address this along with the printf vs wcout which seems related there. I believe this should be addressed in a subsequent update to the project.